packages/org-unit-dialog/src/UserOrgUnitsPanel.js

```diff
@@ -9,39 +10,50 @@ import StopIcon from '@material-ui/icons/Stop';
 import userOrgUnits from './userOrgUnits';
 import styles from './styles/UserOrgUnitsPanel';
 
-const UserOrgUnitsPanel = props => (
-    <div style={styles.container}>
-        <Grid
-            container
-            direction="row"
-            alignItems="center"
-        >
-            {userOrgUnits.map(orgUnitType => (
-                <Grid
-                    key={orgUnitType.id}
-                    style={styles.gridItem}
-                    item
-                >
-                    <Checkbox
-                        checked={props.userOrgUnits.some(ouType => ouType.id === orgUnitType.id)}
-                        onChange={props.handleUserOrgUnitClick}
-                        inputProps={{
-                            id: orgUnitType.id,
-                            name: orgUnitType.id,
-                        }}
-                        icon={<CheckBoxOutlineBlankIcon style={styles.checkbox} />}
-                        checkedIcon={<CheckBoxIcon style={styles.checkboxChecked} />}
-                        color={props.checkboxColor}
-                    />
-                    <InputLabel htmlFor={orgUnitType.id}>
-                        <StopIcon style={styles.stopIcon} />
-                        <span style={styles.text}>{orgUnitType.displayName}</span>
-                    </InputLabel>
-                </Grid>
-            ))}
-        </Grid>
-    </div>
-);
+const UserOrgUnitsPanel = props => {
```

Comment:
This looks like a big change, but the only change is the addition of `orgUnitNames` map, and using the map on line 28. All else is the same as before

Reply:
No worries, good as is. But if you move the `orgUnitNames` const out of the `UserOrgUnitsPanel` it should show up as smaller change.